Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Position_for_footnote_and_endnote
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Creates a new Word document.
+            using (WordDocument document = new WordDocument())
+            {
+				//Creates a section.
+				IWSection section = document.AddSection();
+				//Adds a paragraph to a section.
+				IWParagraph paragraph = section.AddParagraph();
+                //Appends the text to paragraph
+                paragraph.AppendText("First paragraph in First section");
+                //Appends the footnotes as it sets the footnote
+                WFootnote footnote = paragraph.AppendFootnote(FootnoteType.Footnote) as WFootnote;
+                //Sets the footnote character format
+                footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+                //Adds footnote text
+                paragraph = footnote.TextBody.AddParagraph();
+                paragraph.AppendText("Footnote content");
+                //Sets the footnote position
+                document.FootnotePosition = FootnotePosition.PrintImmediatelyBeneathText;
+                //Adds the new section to the document
+                section = document.AddSection();
+                //Adds a paragraph to a section
+                paragraph = section.AddParagraph();
+                //Inserts the text into the paragraph
+                paragraph.AppendText("Paragraph in Second section.");
+                //Appends the endnotes. Sets the footnote or endnote
```

Comment:
@sharonaumao - Please modify this XML comments properly.

Reply:
@Ramaraj-Marimuthu -I made the necessary corrections